OpenEdXMobile/src/main/java/org/edx/mobile/model/db/DownloadEntry.java

```diff
@@ -215,7 +215,7 @@ public TranscriptModel getTranscripts() {
     @Override
     public void setDownloadInfo(NativeDownloadModel download) {
         dmId = download.dmid;
-        downloaded = DownloadedState.DOWNLOADING;
+        downloaded = DownloadedState.values()[(int)download.downloaded];
```

Comment:
@farhan I am not sure how this change got in here. Could this be from a bad conflict resolution?  I am looking at this now.